pkg/model/v1alpha1/storage_source.go

```diff
@@ -15,6 +15,7 @@ type StorageSourceType int
 const (
 	storageSourceUnknown StorageSourceType = iota // must be first
 	StorageSourceIPFS
+	StorageSourceRepoClone
```

Comment:
we should not update previous versions of our schema. They must remain the same to maintain backward compatibility with jobs created with older schema

Reply:
Ok will remove that